ORCID.js

```diff
@@ -0,0 +1,99 @@
+{
+	"translatorID": "e83248bb-caa4-4dd2-a470-11f4cd164083",
+	"label": "ORCID",
+	"creator": "Philipp Zumstein",
+	"target": "https?://orcid\\.org/",
+	"minVersion": "3.0",
```

Comment:
Yes, it should be ready. However, when I try to test it now, it is not working for me.

Is it working for you?